app/main.py

```diff
@@ -1,8 +1,64 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int | float,
+            clean_power: int,
+            average_rating: int | float,
+            count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
```

Comment:
Code Style issue: The 'CarWashStation' class has too many instance variables. This can make the class difficult to maintain. Consider grouping related variables into classes. For example, 'average_rating' and 'count_of_ratings' can be grouped into a 'Rating' class.